src/pages/home/HomePage.js

```diff
@@ -124,6 +125,12 @@ class HomePage extends PureComponent {
                     <RightDockedModal route={ROUTES.SEARCH}>
                         <SearchPage />
                     </RightDockedModal>
+                    <RightDockedModal route={ROUTES.IOU_REQUEST_MONEY}>
+                        <IOUModal />
```

Comment:
Strange to see here that it's the same component for both routes... seems unnecessary, but maybe reading further in the PR will yield the answer.

Reply:
Addressed in the other comment 👍